shell/platform/linux/fl_view.cc

```diff
@@ -255,12 +271,135 @@ static void fl_view_size_allocate(GtkWidget* widget,
   FlutterEngineSendWindowMetricsEvent(self->flutter_engine, &event);
 }
 
+static gboolean fl_view_button_press_event(GtkWidget* widget,
+                                           GdkEventButton* event) {
+  FlView* self = FL_VIEW(widget);
+
+  if (self->flutter_engine == nullptr)
+    return FALSE;
+
+  // Flutter doesn't handle double and triple click events
+  if (event->type == GDK_DOUBLE_BUTTON_PRESS ||
```

Comment:
Are these separate from the individual press events? I.e., if someone double-clicks, will there be two down-up event pairs even if we ignore the double click event?

Reply:
GTK sends double click events in addition to the single click events. So we can just safely ignore them. The harder case is if you want to react to a double click and not a single click, as you get both those events.